### PR TITLE
Add github workflow for violet-demo-app

### PR DESCRIPTION
**Pull request by AndyChinViolet**

### CODE REVIEW

**Comment by AndyChinViolet** [comment]
Current dependencies on/for this PR:
* main
  * **PR #10** <a href="https://app.graphite.dev/github/pr/violetio/VioletReactDemoApp/10" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/violetio/VioletReactDemoApp/10?utm_source=stack-comment).

**Review by sebastian-violet — APPROVED** [review]
I assume that there will only be one stage that will be tagged.
We will want `prod` tag.

File: .github/buildAndDeploy.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  workflow_dispatch:
+    inputs:
+      force-deploy:
+        type: boolean
+        required: true
+        description: "Force Deploy this out to Dev even if it's not the main branch?"
+  push:
+
+jobs:
+  violet-connect:
```
Comment (sebastian-violet) [inline .github/buildAndDeploy.yaml:13]:
```suggestion
  violet-demo-app
```

Comment (AndyChinViolet) [reply .github/buildAndDeploy.yaml:13]: derp

**Review by sebastian-violet — APPROVED** [review]

**Pull request merged**